pkg/virt-controller/services/template.go

```diff
@@ -199,6 +240,10 @@ func CPUFeatureLabelsFromCPUFeatures(vmi *v1.VirtualMachineInstance) []string {
 	return labels
 }
 
+func KVMInfoHypervSupportLabel() string {
+	return NFD_KVM_INFO_PREFIX + "hyperv"
```

Comment:
just curious: why a func for this, and not just a const?

Reply:
> just curious: why a func for this, and not just a const?

No real reason. Let me simplify.